startup.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Install HomeBrew if not installed 
+if ! command -v brew >/dev/null 2>&1
+then   
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+else
+    echo "Home brew is installed"
+fi 
+
+# Install jq if not installed
+if ! command -v jq >/dev/null 2>&1
+then
+    brew install jq 
+else
+    echo "JQ is installed"
+fi 
+
+# Install Twilio CLI if not installed 
+if ! command -v twilio >/dev/null 2>&1
+then
+    brew tap twilio/brew && brew install twilio
+else
+    echo "Twilio cli is installed"
+fi 
+
+# Set Credentials for making Twilio API calls
+while getopts ":a:b:c:" OPTION 
```

Comment:
Can we add a usage() function to prompt users on expected values for a, b, c here? Not a requirement for the script to run, but a good practice & "a b c" arent really mapped to anything memorable compared to what they are expecting:
- https://stackoverflow.com/questions/34567826/what-is-usage-in-shell-scripting